### PR TITLE
refactor the rabbitmqctl command calls

### PR DESCRIPTION
**Pull request by alexconrey**

### CODE REVIEW

**Review by miguelwhite — DISMISSED** [review]
LGTM

File: lib/puppet/provider/rabbitmq_policy/rabbitmqctl.rb [hunk 1]
```diff
@@ -5,48 +5,55 @@
...
+    end
+
+    json_policies = JSON.parse(all_policies)
+
+    if json_policies.empty?
+      #puts "empty policy list"
+      return
+    else
+      json_policies.each do |policy|
+        marshal_policy = policy
```
Comment (alexconrey) [inline lib/puppet/provider/rabbitmq_policy/rabbitmqctl.rb]: unused?

File: lib/puppet/provider/rabbitmq_user_permissions/rabbitmqctl.rb [hunk 2]
```diff
@@ -10,22 +10,56 @@
...
+
+    all_users.gsub!('ok', '')
+
+    json_users = JSON.parse(all_users)
+
+    if json_users.empty?
+      return
+    else
+      json_users.each do |user|
+        puts user
```
Comment (alexconrey) [inline lib/puppet/provider/rabbitmq_user_permissions/rabbitmqctl.rb]: delete this

**Review by saltoilmsg — APPROVED** [review]
lgtm - lots to take in...

**Pull request merged**